src/renderers/shared/hooks/ReactChildrenMutationWarningHook.js

```diff
@@ -46,21 +44,13 @@ function handleElement(debugID, element) {
   );
 }
 
-var ReactDOMUnknownPropertyHook = {
-  onBeforeMountComponent(debugID, element) {
-    elements[debugID] = element;
-  },
-  onBeforeUpdateComponent(debugID, element) {
-    elements[debugID] = element;
-  },
-  onComponentHasMounted(debugID) {
-    handleElement(debugID, elements[debugID]);
-    delete elements[debugID];
```

Comment:
For future reference: `delete` is super slow on V8 with many items when using numeric keys. Even `toString()` is not enough—you have to prepend them with something like `.`.